whylog/config/filename_parsers.py

```diff
@@ -0,0 +1,17 @@
+from abc import ABCMeta, abstractmethod
+
+
+class AbstractFilenameParser(object):
```

Comment:
I suggest to change the name to `AbstractFilenameMatcher` so that it is harder to mistake it with the hierarchy of `Parsers`